_posts/2020-09-02-cockpit-227.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Cockpit 227
+author: pitti
+date: 2020-09-02
+tags: cockpit linux
+slug: cockpit-227
+category: release
+summary: Cockpit with VM filtering and PatternFly 4 design changes
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 227.
+
+### Machines: VM list filtering
+
+The Virtual Machine overview page can now be filtered by name and/or by state.
```

Comment:
```suggestion
The Virtual Machine overview page can now be filtered by name and state.
```

https://en.wikipedia.org/wiki/And/or

While technically not wrong, it's a bit messy and generally informal (although it has been used in super-formal legal documents, so this does not necessarily hold). In this case, we can probably just shorten it to "and", as it can be filtered by both. We could use "or" instead, but it may imply that the two filters are mutually exclusive (using one at a time).